ctapipe/io/hessiofilereader.py

```diff
@@ -20,10 +19,6 @@ class HessioFileReader(EventFileReader):
     """
     _count = 0
 
-    input_url = Unicode(
```

Comment:
Removing this removes the ability to set a default path for the reader. I think that is useful functionality to keep.

Reply:
it was just confusing as it creates another attribute called input_url that shadows the parent class attribute, which is generally bad practice. We could just have a default value attribute that gets filled in if input_url is blank.  I don't really see having a default is very useful, though. The input shoudl be explicitly specified.

Reply:
Okay, lets discard defaults